### DISCUSSION TITLE
Disabled BC instnace restarts when using ImportTestDataInBcContainer

### DISCUSSION THREAD

**Discussion by gntpet** [microsoft/AL-Go#1571] [Q&A] [answered]
Hey there, 

When using custom ImportTestDataInBcContainer, I still see that pipeline triggers server restart, which is not needed in our case. Would it be possible to disable it?

![image](https://github.com/microsoft/AL-Go/assets/16342128/5f90e173-81a8-4e70-b649-0d227bff1466)

My custom ImportTestDataInBcContainer looks like this:
```
Param([Hashtable]$parameters) 
New-CompanyInBcContainer  -containerName $parameters.containerName -tenant $parameters.tenant -companyName "TNM Automated Test"
Invoke-NavContainerCodeunit -containerName $parameters.containerName -tenant $parameters.tenant -CompanyName "TNM Automated Test" -Codeunitid 2 6> $null    
```

**Comment by freddydk** [comment, marked as answer]
If you set EnableTaskScheduler to true in your repository or project settings, then it shouldn't restart the service tier to enable the task scheduler.
